### PR TITLE
fixed cron job interpretation in the example

### PR DESCRIPTION
**Pull request by lintopaul**
the cron job in this example runs on the intersection of 13th of every month and Friday, not the union of these conditions. If the 13th falls on any other day of the week, the job will not run.

### Issue

Closes: #

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ The `.spec.schedule` field is required. The value of that field follows the [Cro
 # * * * * *
 ```
 
-For example, `0 0 13 * 5` states that the task must be started every Friday at midnight, as well as on the 13th of each month at midnight.
+For example, `0 0 13 * 5` means this cron job will run at midnight(12:00 AM) on Friday the 13th of every month.
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/controllers/cron-jobs.md:68]: This is a weird crontab spec. The date has to be a Friday, and it has to be the 13th day of a month ... emm.
I'd suggest we change this to something more realistic.

**Review by lintopaul — COMMENTED** [review]
replaced example with something more realistic

File: content/en/docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ The `.spec.schedule` field is required. The value of that field follows the [Cro
 # * * * * *
 ```
 
-For example, `0 0 13 * 5` states that the task must be started every Friday at midnight, as well as on the 13th of each month at midnight.
+For example, `0 0 1,10,20 * 1,3,5` means this cron job is scheduled to run at midnight on the 1st, 10th, and 20th of each month, but only if those days fall on a Monday, Wednesday, or Friday.
```
Comment (nate-double-u) [inline content/en/docs/concepts/workloads/controllers/cron-jobs.md:68]: This is a complex crontab too -- requiring both a specific set of dates and days of the week.

Could we use something that triggers weekly on a Monday at 3am? I don't think this example needs to be super in depth, folks can go to the cron docs for complex cron setup.

**Comment by tengqm** [comment]
This new cron expression is much better than the existing one.
/lgtm
/label tide/merge-method-squash

File: content/en/docs/concepts/workloads/controllers/cron-jobs.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ The `.spec.schedule` field is required. The value of that field follows the [Cro
 # * * * * *
 ```
 
-For example, `0 0 13 * 5` states that the task must be started every Friday at midnight, as well as on the 13th of each month at midnight.
+For example, `0 3 * * 1` means this cron job is scheduled to run weekly on a Monday at 3 AM.
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/cron-jobs.md:68]: OK, but keep the “task“ part:
```suggestion
For example, `0 3 * * 1` means this task is scheduled to run weekly on a Monday at 3 AM.
```

**Review by nate-double-u — COMMENTED** [review]
/lgtm

**Comment by sftim** [comment]
Thanks!

/approve

**Pull request merged**